beginner_source/flava_finetuning_tutorial.py

```diff
@@ -0,0 +1,190 @@
+# -*- coding: utf-8 -*-
+"""
+TorchMultimodal Tutorial: Finetuning FLAVA
+============================================
+"""
+
+######################################################################
+# Multimodal AI has recently become very popular owing to its ubiquitous
+# nature, from use cases like image captioning and visual search to more
+# recent applications like image generation from text. **TorchMultimodal
+# is a library powered by Pytorch consisting of building blocks and end to
+# end examples, aiming to enable and accelerate research in
+# multimodality**.
+# 
+# In this tutorial, we will demonstrate how to use a **pretrained SoTA
+# model called** `FLAVA <https://arxiv.org/pdf/2112.04482.pdf>`__ **from
+# TorchMultimodal library to finetune on a multimodal task i.e. visual
+# question answering** (VQA). The model consists of two unimodal transformer
+# based encoders for text and image and a multimodal encoder to combine
+# the two embeddings. It is pretrained using contrastive, image text matching and 
```

Comment:
Can the losses be enumerated in a different way here? I feel the comma placement makes this kinda confusing